app/controllers/cases/filters_controller.rb

```diff
@@ -82,6 +83,7 @@ def my_open
           :responding_team,
           :approver_assignments
         )
+
       service = call_search_service(unpaginated_cases, cookies[:search_result_order])
       @query = service.query
 
```

Comment:
you can set the current active tab's count here like 

`@global_nav_manager.current_page_or_tab.set_count(service.result_set.count)`